src/generate/index.js

```diff
@@ -30,16 +32,18 @@ function emit (type, status, attrs) {
   }, attrs))
 }
 
-function parseCountries (countries) {
+function parseCountries (locations) {
   emit('countries', 'start')
-  return csv.parseAsync(countries, {
+  return csv.parseAsync(locations, {
     columns: true,
     cast: false,
     skip_empty_lines: true
   })
     .then((parsed) => {
       return _.reduce(parsed, (acc, row) => {
-        acc[row.alpha2] = normalizeName(row.name)
+        if (typeof acc[row.country_iso_code] === 'undefined') {
```

Comment:
📝: I personally like following instead

```suggestion
        if (typeof acc[row.country_iso_code] != null) {
```
ensures that it's neither of non values.

Reply:
@Gozala sgtm, but fyi eslint does not like this,  had to apply `// eslint-disable-line  valid-typeof`